google_api_availability_platform_interface/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
+
+- Initial release.
```

Comment:
Can we put something more meaning full here (and following the [Google Style guide](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog))? Maybe something a long the lines of:
```suggestion
- Extracts the common platform interface from the google_api_availability package.
```